airbyte-cdk/python/pyproject.toml

```diff
@@ -24,7 +24,7 @@ keywords = ["airbyte", "connector-development-kit", "cdk"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-airbyte-protocol-models = "*"
+airbyte-protocol-models = "~0.9"
```

Comment:
If you want to keep it under 1.0, but are okay with 0.10, 0.11 etc, this should be `^0.9.0`

Reply:
... or [that might be wrong specifically for 0 versions](https://python-poetry.org/docs/dependency-specification/), now i'm confused

Reply:
I guess we can do `>= 0.9.0, < 2.0`

Reply:
You mean <1.0? Works!

Reply:
yes... 7 commits for a 1-line change 🤦